exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -132,41 +151,44 @@ private void AskQuestion()
 
         private string CurrentCategory()
         {
-            if (places[currentPlayer] == 0) return "Pop";
-            if (places[currentPlayer] == 4) return "Pop";
-            if (places[currentPlayer] == 8) return "Pop";
-            if (places[currentPlayer] == 1) return "Science";
-            if (places[currentPlayer] == 5) return "Science";
-            if (places[currentPlayer] == 9) return "Science";
-            if (places[currentPlayer] == 2) return "Sports";
-            if (places[currentPlayer] == 6) return "Sports";
-            if (places[currentPlayer] == 10) return "Sports";
-            return "Rock";
+            return places[_currentPlayer] switch
```

Comment:
I like pattern matching. It's probably the next best thing to one of [these](https://refactoring.guru/refactoring/techniques/simplifying-conditional-expressions). Again quite a new feature so don't be surprised if people haven't seen it before.